src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -240,11 +238,11 @@ namespace AirflowNetwork {
                 }
             }
         }
-        if (allocated(m_state.dataZoneEquip->ZoneEquipConfig) && m_state.dataHVACGlobal->NumHybridVentSysAvailMgrs > 0 &&
+        if (allocated(m_state.dataZoneEquip->ZoneEquipConfig) && m_state.dataAvail->NumHybridVentSysAvailMgrs > 0 &&
             allocated(m_state.dataAirSystemsData->PrimaryAirSystems)) {
             hybrid_ventilation_control();
         }
-        if (VentilationCtrl == 1 && m_state.dataHVACGlobal->NumHybridVentSysAvailMgrs > 0) {
+        if (ventCtrlStatus == Avail::VentCtrlStatus::Open && m_state.dataAvail->NumHybridVentSysAvailMgrs > 0) {
```

Comment:
This was a problem even before the conversion to an enumeration. There was a `constexpr int HybridVentCtrl_Open` which should have been used instead. Changing to an enumeration forces us to use the symbol instead of the literal.

Reply:
Question: if NumHybridVentSysAvailMgrs == 0, how can ventCtrlStatus == Avail::VentCtrlStatus::Open ?

Reply:
Don't look at me, I just search and replace.